### PR TITLE
Adds old hippie space funeral mobs

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
-adds philip, leghorse, bloodhound and bloodflea

[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: philip, leghorse, bloodhound and bloodflea are now in the game, prepare to cry

/:cl:

## About The Pull Request
y'all remember how space funeral 3 is made by furries and is gay? shit's mad

## Why It's Good For The Game
ALLAH

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/retaliate/spacefuneral.dm [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/mob/living/simple_animal/hostile/space_funeral
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/hostile/retaliate/spacefuneral.dm:1]: I don't think this is supposed to be its own separate creature.

File: code/modules/mob/living/simple_animal/hostile/retaliate/spacefuneral.dm [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	name = "bloodhound"
+	desc = "A sanity-destroying dog-like creature."
+	icon = 'icons/mob/spacefuneral.dmi'
+	speak = list("gibbers")
+	icon_state = "bloodhound"
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/hostile/retaliate/spacefuneral.dm]: delete everything from `icon_state` to `attack_sound`

File: code/modules/mob/living/simple_animal/hostile/retaliate/spacefuneral.dm [hunk 3]
```diff
@@ -0,0 +1,73 @@
+/mob/living/simple_animal/hostile/space_funeral
+	name = "bloodhound"
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/hostile/retaliate/spacefuneral.dm]: change name and description to something more reasonable

File: code/modules/mob/living/simple_animal/hostile/retaliate/spacefuneral.dm [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+	speak = list("gibbers")
+	health = 1
+	maxHealth = 1
+	melee_damage_lower = 1
+	melee_damage_upper = 2
+	attacktext = "pokes"
+	attack_sound = 'sound/weapons/bite.ogg'
+	faction = list("funeral")
+
+
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/hostile/retaliate/spacefuneral.dm:6]: fix the number of spaces between each mob and compile to see if it works

**Comment by karmaisblackandbluepilled** [comment]
tested ingame, fixed number of lines, here are the kings
Capture

**Comment by HideAndSeekLOGIC** [comment]
SInce it's an old hippie port I'll just merge straight away

**Comment by Manly-Dwarf** [comment]
>SPEEDMERGING THESE MOBS

OH NO NO NO NO NO OH GOD
WE KNOW EXACTLY WHAT HAPPENED LAST TIME SOUND THE FUCKING ALARM

**Pull request merged**